VersionControlUI/src/DiffManager.h

```diff
@@ -141,12 +149,20 @@ class VERSIONCONTROLUI_API DiffManager
 
 		bool processNameChange(Model::Node* oldNode, Model::Node* newNode, const DiffSetup& diffSetup);
 
-		void showNameChangeInformation(Visualization::ViewItem* currentViewItem, const DiffSetup& diffSetup);
+		void showNameChangeInformation(Visualization::ViewItem* currentViewItem, const DiffSetup& diffSetup,
+												 VersionControlUI::DiffComparisonPair* diffComparisonPair);
+
+		bool satisfiesTargetNodeIdConstraint(Model::Node* oldNode, Model::Node* newNode,
```

Comment:
I don't understand the names of this method and the next one. What do they do? Can we come up with better names?